packages/fx-core/templates/azure/config.bicep

```diff
@@ -0,0 +1,5 @@
+@secure()
```

Comment:
are `packages/fx-core/templates/azure/xxx` test files? we may need to add cleanup code for test files, in the meanwhile, we may set project root to somewhere under test folder

Reply:
seems solution.scaffold.tests.ts create this test folder, already removed